MANIFEST.in

```
@@ -3,4 +3,5 @@ graft docs
 prune docs/build
 graft tests
 include readme_pypi.rst
+recursive-include requirements *.txt
```

```suggestion
graft requirements
```
Good catch 🎉  but should we just get the whole dir instead?

---

On second thought, given that the requirements folder is only useful for the development from source, this needs to be reevaluated. I'll give it some consideration the next time I have a minute.

---

The manifest includes documentation and tests, so it would make sense to include requirements to build docs and run tests. Or did I misunderstand?

---

The manifest was fixed a bunch in another commit, this is no longer needed.